pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -92,6 +94,7 @@
 , debug ? false
 , developerBuild ? false
 , qttranslations ? null
+, isCrossBuild ? stdenv.hostPlatform != stdenv.buildPlatform
```

Comment:
Should be `stdenv.buildPlatform.canExecute stdenv.hostPlatform` and be kept in the let .. in clause below (as there's no point in exposing this as an argument).

Reply:
Fixed